rack_content_type_default.gemspec

```diff
@@ -1,26 +1,26 @@
-lib = File.expand_path('../lib/', __FILE__)
-$:.unshift lib unless $:.include?(lib)
+# frozen_string_literal: true
 
-require 'rack/content_type_default/version'
+require_relative 'lib/rack/content_type_default/version'
 
 Gem::Specification.new do |s|
-  s.name                      = 'rack_content_type_default'
-  s.version                   = Rack::ContentTypeDefault::VERSION
-  s.authors                   = ['Purnima Mavinkurve', 'Connor Savage']
-  s.email                     = ['pmavinkurve@mdsol.com', 'csavage@mdsol.com']
-  s.homepage                  = 'https://github.com/pmavinkurve-mdsol/rack_content_type_default'
-  s.summary                   = 'Rack Middleware for setting content type when not provided' 
-  s.description               = 'Rack Middleware for setting content type when not provided'
+  s.name                  = 'rack_content_type_default'
+  s.version               = Rack::ContentTypeDefault::VERSION
+  s.authors               = ['Purnima Mavinkurve', 'Connor Savage']
+  s.email                 = ['pmavinkurve@mdsol.com', 'csavage@mdsol.com']
```

Comment:
How about we put your name and email @jfeltesse-mdsol instead of Connor's? Thanks.

Reply:
I've removed Connor's email because it is now invalid but I left his name to keep the credits where they're due (who knows, maybe Connor will brag about this gem in his future endeavours? 😄)

I didn't add my name because I don't qualify as an author, I didn't even look at the code TBH. I simply added the necessary bits for publishing and cleaned up a bit, that's it 😅

Reply:
Sounds good. Thanks @jfeltesse-mdsol !